readme.md

```diff
@@ -54,6 +54,13 @@ $ cd SpoofDPI
 $ go build ./cmd/...
 ```
 
+## docker
+You can use Dockerfile to build a docker image
+```shell
+docker build ./ -t spoof-dpi:latest
+```
+Use `docker run -it -p 8080:8080 --name spoof-dpi spoof-dpi:latest` to run container
```

Comment:
i think we should show some other example that passes options through the docker run command like below
```bash
$ docker run -it -p 8080:8080 spoof-dpi:latest --enable-doh
```

Reply:
@xvzc Better also add `--name "..."` and `--rm` to avoid generating a lot of garbage after each restart. 
https://docs.docker.com/reference/cli/docker/container/run/#rm